py/misc.h

```diff
@@ -52,6 +52,11 @@ typedef unsigned int uint;
 
 // Static assertion macro
 #define MP_STATIC_ASSERT(cond) ((void)sizeof(char[1 - 2 * !(cond)]))
+#if defined(_MSC_VER)
+#define MP_STATIC_ASSERT_NOT_MSC(cond)
```

Comment:
I think this needs to be `#define ... (1)`.

Reply:
Probably. (I had in mind that this is fine because many `assert` impls are preprocessed to empty, but this makes sense). Changing